services/power.md

```diff
@@ -4,6 +4,35 @@
 
 A power-provider service.
 
+## Power negotiation protocol
+
+The purpose of the power negotiation is to ensure that there is no more than ~500mA
+delivered to the power rail.
+This is realized by limiting the number of enabled power provider services to 1.
+
+Note, that it's also possible to have low-current power providers, which do not
+run a power provider service.
+These are **not** accounted for in the power negotiation protocol.
+
+* Upon reset, a power service enables itself.
+* Every enabled power service emits power `on` reports with its announce packets.
```

Comment:
- wall vs battery: should we prioritize wall power from battery power?
- battery level: should we always pick the highest battery leve possible?

Reply:
the power service often wouldn't know the power source or capacity (it may be just a microUSB -> JACDAC "converter", which has no insight into where the power is coming from).

Still, I guess we could have some notion of priority.

Reply:
If we envision power module with built-in lipo+charging circuit, one would expect those modules to receive power while developing (e.g. brain is connected to computer powering the bus) and give power when the brain is disconnected.

Reply:
OK, not doing this in this service